pkg/tsdb/cloudwatch/request_parser.go

```diff
@@ -37,7 +37,7 @@ type QueryJson struct {
 	Statistic         *string                `json:"statistic,omitempty"`
 	Statistics        []*string              `json:"statistics,omitempty"`
 	TimezoneUTCOffset string                 `json:"timezoneUTCOffset,omitempty"`
-	QueryType         string                 `json:"queryType,omitempty"`
+	QueryType         string                 `json:"type,omitempty"`
```

Comment:
Could we add tests to protect this key for the future?

Reply:
added!